content/docs/setup-and-administration/configuring-app-registrations-for-non-interactive-use/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Configuring App Registrations for Non-Interactive Use"
+description: "Creating App Registrations for RKVST"
+lead: "Creating App Registrations for RKVST"
+date: 2021-06-16T11:12:25+01:00
+lastmod: 2021-06-16T11:12:25+01:00
+draft: false
+images: []
+menu: 
+  docs:
+    parent: "setup-and-administration"
+weight: 12
+toc: true
+---
+
+Non-interactive access to the RKVST platform is managed by creating `App Registrations` either via the Manage RKVST Interface in the UI or by using the App Registrations API directly.
+
+`App Registrations` are Authentication Profiles with a `CLIENT ID` and `SECRET` that can then be used to generate JWT Tokens for authenticating to RKVST API Endpoints.
```

Comment:
Important: AFAIK the API is called "App Registrations" but the resources we're managing are "Applications". I'd check this with @JAG-UK but its the messaging I've had previously, and it's reflected in the API design.

Reply:
I'm not fussed I'll do it however - @JAG-UK which do you prefer where?

Reply:
Resolved